zips/zip-0234.md

```diff
@@ -164,58 +146,63 @@ The deployment should happen at the next halving, which is block `2726400`.
 
 Since there is a planned halving at this point, there will already be a
 significant "shock" caused by the drop in issuance caused by the halving. This
-reduces surprise and thus increases security. Also, due to the nature of the
-smoothed curve having a portion of the curve above the respective step function
-line at times, this will maximally _reduce_ the issuance shock at the
-`DEPLOYMENT_BLOCK_HEIGHT`.
+reduces surprise and thus increases security, satisfying **Requirement 5**.
+Also, due to the nature of the smoothed curve having a portion of the curve
+above the respective step function line at times, this will maximally _reduce_
+the issuance shock at the `DEPLOYMENT_BLOCK_HEIGHT`.
```

Comment:
This depends upon the activation height; this is only true if this activates at a halving.